### ISSUE TITLE
Tutorial: Plot additional features (coastlines, symbols, text) on top of a topographic surface

### ISSUE THREAD

**Issue by yvonnefroehlich** [GenericMappingTools/pygmt#3429]
For context see:
- https://github.com/GenericMappingTools/pygmt/pull/3316/files#r1707195092
- https://github.com/GenericMappingTools/pygmt/pull/3316#discussion_r1730713006

Besides plotting a dataset (grid or image) on top of a topographic surface, one may want to add additional features like coastlines, symbols, and text. We can 
- expand the tutorial "Draping a dataset on top of a topographic surface".
- create a standalone tutorial on this.

**Comment by seisman** [comment]
> * create a standalone tutorial on this.

We should do this

**Comment by yvonnefroehlich** [comment]
> > * create a standalone tutorial on this.
> 
> We should do this

Yes, also think, that this is better, so we can focus on one task per tutorial.
